core/scss/components/drawer/drawer.scss

```diff
@@ -0,0 +1,63 @@
+@include exports('md-drawer') {
+
+  .#{$prefix}-drawer {
+    z-index: $drawer-zindex;
+    position: fixed;
+    overflow: auto;
+    transition-duration: 0.6s;
+    &.#{$prefix}-drawer__right {
+      width: 0px;
+      top: 0px;
+      right: 0;
+      height: 100vh;
+      margin-right: auto;
+      margin-left: auto;
+    }
+    &.#{$prefix}-drawer__top, &.#{$prefix}-drawer__bottom   {
+      height: 0px;
+      width: 100%;
+    }
+    &.#{$prefix}-drawer__top {
+      top: 0;
+    }
+    &.#{$prefix}-drawer__bottom {
+      bottom: 0;
+     // top: unset;
+    }
+    &.#{$prefix}-drawer__open {
+      &.#{$prefix}-drawer__right {
+        &.#{$prefix}-drawer--small {
+          width: 200px;
+        }
+        &.#{$prefix}-drawer--medium {
+          width: 535px;
+        }
+        &.#{$prefix}-drawer--large {
+          width: 800px;
+        }
+      }
+      &.#{$prefix}-drawer__top, &.#{$prefix}-drawer__bottom {
+        &.#{$prefix}-drawer--small {
+          height: 200px;
+        }
+        &.#{$prefix}-drawer--medium {
+          height: 535px;
+        }
+        &.#{$prefix}-drawer--large {
+          height: 800px;
+        }
+
+      }
+      background-color: $drawer-background-color;
+      border: 1px solid $drawer-border-color;
+      box-shadow: 0px 1px 2px 1px rgba(52, 53, 55, 0.18);
+      @include vr-spacing(p, 1);
+    }
+    #{$prefix}-icon {
```

Comment:
We would want to add a class to icon and target it so that if an adopter uses an icon within the drawer it isn't affected.  This would be like md-drawer__close-icon